features/admin.userstores.v1/components/user-stores-list.tsx

```diff
@@ -279,10 +283,14 @@ export const UserStoresList: FunctionComponent<UserStoresListPropsInterface> = (
                 return (
                     <EmptyPlaceholder
                         action={ (
-                            <PrimaryButton onClick={ onEmptyListPlaceholderActionClick }>
-                                <Icon name="add" />
-                                { t("userstores:placeholders.emptyList.action") }
-                            </PrimaryButton>
+                            disabledFeatures?.includes("userStores.type.remote")
```

Comment:
Let's use `isFeatureEnabled` from @wso2is/core and define the feature identifier in the FEATURE_DICTIONARY, 

similar impl: https://github.com/wso2/identity-apps/blob/master/features/admin.applications.v1/constants/application-management.ts#L68

Reply:
Improved with dbeaf64e238a32b7625cab8eab1a24518f1ff4d4.